lib/bibliothecary/parsers/maven.rb

```diff
@@ -148,6 +152,16 @@ def self.parse_maven_resolved(file_contents)
           .compact
           .uniq
       end
+      def self.parse_maven_tree(file_contents)
+        captures = file_contents.scan(/\s([\w\.-]+:[\w\.-]+:[\w\.-]+:[\w\.-]+:[a-zA-Z]+)/).flatten.uniq
```

Comment:
I'm just gonna go ahead and trust this regex is correct ;)

Reply:
Should be able to replace the `\.`'s with `.` since you don't need to escape in character groups.